src/switcher-ui.tsx

```
@@ -721,13 +761,15 @@ function SwitcherApp() {
     if (key === extraPinnedKeyRef.current) return;
     extraPinnedKeyRef.current = key;
     if (missing.length === 0) {
+      extraPinnedSessionsRef.current = [];
       setExtraPinnedSessions([]);
       return;
     }
     window.electronAPI.getSessionsByIds(missing).then((result: any[]) => {
```

ESLint

[error] Replace `.getSessionsByIds(missing)` with `⏎······.getSessionsByIds(missing)⏎······`

(prettier/prettier)